lib/src/main/java/com/basistheory/android/service/BasisTheoryElements.kt

```diff
@@ -51,11 +54,24 @@ class BasisTheoryElements internal constructor(
 
     @JvmOverloads
     suspend fun createSession(apiKeyOverride: String? = null): CreateSessionResponse =
-        withContext(ioDispatcher) {
+        withContext(dispatcher) {
             val sessionsApi = apiClientProvider.getSessionsApi(apiKeyOverride)
             sessionsApi.create()
         }
 
+    @JvmOverloads
+    suspend fun getToken(
+        id: String,
+        apiKeyOverride: String? = null
+    ): Token =
+        withContext(dispatcher) {
+            val tokensApi = apiClientProvider.getTokensApi(apiKeyOverride)
+
+            val token = tokensApi.getById(id)
+            token.data = transformResponseToValueReferences(token.data)
+            token
```

Comment:
This might be cleaner as 
```suggestion
            tokensApi.getById(id).also {
                it.data = transformResponseToValueReferences(token.data)
            }
```